docs/components/modeler/web-modeler/camunda-marketplace.md

```diff
@@ -1,25 +1,29 @@
 ---
 id: camunda-marketplace
-title: Visit the Camunda Marketplace
-description: "Visit the Camunda Marketplace to browse and filter all available Connectors, and incorporate them into your BPMN diagram."
+title: Camunda Marketplace
+description: "Visit the Camunda Marketplace to browse available resources, and incorporate them into your project."
 ---
 
-While modeling a process, you can click into the Camunda Marketplace to browse through Connectors provided by Camunda engineering, discover Connectors developed by Camunda partners, and explore Connectors developed by the Camunda community.
+Discover the **Camunda Marketplace**, your go-to destination for leveraging various contributions from the Camunda community, trusted partners, and the Camunda team.
 
-This may ease your process of selecting which Connectors to integrate into your process, as you can filter by creator, [inbound versus outbound Connectors](/docs/components/connectors/connector-types.md), [runtime](/docs/components/connectors/introduction.md), and Camunda versions that include specific out-of-the-box Connectors.
+Follow our intuitive guides to explore and harness Connectors and process blueprints using Web Modeler. If you prefer to utilize these resources within Desktop Modeler, download them directly from the [Camunda Marketplace website](https://marketplace.camunda.com).
 
-:::note
-The Camunda Marketplace can be accessed via your [browser](https://marketplace.camunda.com/) and directly from Web Modeler while building BPMN diagrams. This guide steps through accessing the Camunda Marketplace **directly from your BPMN diagram**. Therefore, screenshots and filters in this guide may appear slightly different from the browser-accessed Camunda Marketplace (though there are no functional differences).
-:::
+If you are a **[Web Modeler Self-Managed](/docs/self-managed/modeler/web-modeler/installation.md)** user, be aware that your organization may restrict access to marketplace resources. If you are unsure about your organization's access, contact your organization's owner for clarification.
 
-## Visit the Camunda Marketplace
+## Browse Marketplace Connectors
 
 To navigate to the Camunda Marketplace, take the following steps:
 
 1. Log in to your Camunda account, and navigate to Web Modeler using the **Camunda components** icon in the top left corner of your console. Click **Modeler**.
```

Comment:
I forgot we are automatically taken to Modeler rather than Console when we log in to camunda.io -- adjusting in a subsequent PR.